examples/pom.xml

```diff
@@ -48,24 +48,23 @@
     </profile>
     <profile>
       <id>release</id>
+      <activation>
+        <property>
+          <name>!quick</name>
```

Comment:
If someone performing a release will specify `-Dquick` wouldn't that be human error? The same is true if they would specify any other flags which they should not.
Regarding the second question, this change indeed makes all the verifications enabled by default and disabled only with `-Dquick`.